tests/test_ind_indygo.py

```diff
@@ -36,33 +36,27 @@ def test_title():
 
 
 def test_description():
-    assert (
-        parsed_items[0]["description"]
-        == "– Board of Finance Meeting following immediately after the Board of Directors meeting."  # noqa
-    )
+    assert parsed_items[0]["description"] == ""
 
 
 def test_start():
-    assert parsed_items[0]["start"] == datetime(2023, 1, 26, 17, 0)
+    assert parsed_items[0]["start"] == datetime(2024, 1, 25, 11, 0)
 
 
 # def test_end():
 #     assert parsed_items[0]["end"] == datetime(2019, 1, 1, 0, 0)
 
 
 def test_time_notes():
-    assert (
-        parsed_items[0]["time_notes"]
-        == "Board meetings are set for 5:00PM unless otherwise noted in meeting description. Please double check the website before the meeting date."  # noqa
-    )
+    assert parsed_items[0]["time_notes"] == ""  # noqa
```

Comment:
See this [comment](https://github.com/City-Bureau/city-scrapers-indianapolis/pull/14/files#r1463728936), re: suppressing linter issues.